### DISCUSSION TITLE
I am using yarn wagmi init in my existing nextjs project

### DISCUSSION THREAD

**Discussion by itsanishjain** [wevm/wagmi#2449] [Q&A]
`yarn wagmi init`

yarn run v1.22.19
$ /home/anish/Documents/projectx/node_modules/.bin/wagmi init

Cannot find module '/home/anish/Documents/projectx/node_modules/@wagmi/cli/node_modules/abitype/dist/esm/zod/index.mts' imported from /home/anish/Documents/freelance/pulsechain/node_modules/@wagmi/cli/dist/commands-X2PX73GF.js
Did you mean to import abitype/dist/cjs/zod/index.js?
error Command failed with exit code 1.

Any idea how to fix this?

**Comment by tmm** [comment]
Related to https://github.com/wagmi-dev/wagmi/issues/2450
